services/worker-manager/src/data.js

```diff
@@ -337,13 +337,32 @@ class Worker {
     const fetchResults = async (query) => {
       const { continuationToken, rows } = await paginateResults({
         query,
-        fetch: (size, offset) =>
-          db.fns.get_queue_workers_with_wm_join(
+        fetch: (size, offset) => {
+          if (Object.keys(query).includes('workerState')) {
```

Comment:
we should probably put this into endpoint description that you can only use one at a time to filter value: either state or qurantined

Reply:
Ah true, good call. Thanks. I was thinking of only _our_ use case which is directly through the UI where only one filter is selectable in the first place. Will update.

Reply:
It's actually already in the docs :) https://firefox-ci-tc.services.mozilla.com/docs/reference/core/worker-manager/api#listWorkers